### PR TITLE
Update time off docs

### PR DESCRIPTION
**Pull request by mpetrovich**

### CODE REVIEW

File: time off & working remotely.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-For example:
+- Always update the travel calendar with any times that you'll be out or working remotely. It's helpful to include contact & availability information too.
 
-* Samir in FL - find me on Slack or call 954-614-XXXX
-* DZ WFH - 347-554-XXXX
-* David - vacation - unreachable
-* Britnee - WF125
+- If you'll be out for a few hours or will be working remotely, post a message in the *nbs-ooo* Slack channel. Use **@here** to ensure that others see your message.
+
+- If you're going to be out for a whole day or more, email everyone an appropriate amount of time in advance.
```
Comment (julienbenatar) [inline time off & working remotely.md]: Can we specify an email subject pattern to help with email filtering? Potentially bringing back `[NBS-OOO]` ?

Comment (mpetrovich) [inline time off & working remotely.md]: @mindweather @dodeca Can we set up something like nbs-ooo@pandora.com?

Comment (dodeca) [inline time off & working remotely.md]: @mindweather I don't have permission to create new groups in the List Manager, do you? https://www.dropbox.com/s/fqvwsdzmmfb5vvk/Screenshot%202015-12-07%2018.35.26.png?dl=0

Comment (mindweather) [inline time off & working remotely.md]: NBS-OOO is back in action, thanks @mpetrovich!

**Pull request merged**